### DISCUSSION TITLE
I am not able to apply for Octernships.

### DISCUSSION THREAD

**Discussion by tamaraiselvan** [community/community#50378] [GitHub Education]
### Select Topic Area

Show & Tell

### Body

I am a student from India, I have GitHub student developer pack. But if I apply to Octernships, It is showing the following message.

"Maximum application limit reached! You cannot apply to this octernship at this moment.", but it is showing like this down the search box.
"12 projects available. You can still apply to 4". What should i do to apply for the octernship.

**Comment by yuriy-o** [comment]
In your case, it seems that you have already applied for 8 projects and now you cannot apply for additional projects until your previous applications are reviewed.

To apply for a new project on Octernships, you need to wait for your previous application for one of the projects to be reviewed first. If your application is accepted, you will be able to continue applying for other projects. If your application is rejected, you will also be able to continue applying to other projects.

In general, you just need to wait for the response to your previous application before submitting a new application. I wish you every success in your efforts!

**Reply by tamaraiselvan** [reply]
I applied for only one, But I did not get the acknowledgement like, I have applied for that, Even thought I applied again if I click that Octernship still apply button is available.

**Reply by hathik66** [reply]
It is showing that i need to contribute more github community to apply for the octornship project.Is there any to get that

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @tamaraiselvan ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by erik-lance** [comment]
I had the same error when applying to certain octernships. Apparently, it was because the number of applicants have reached the max limit for that octernship. I tried on a different one that I was interested in and it worked. I think the limit is 500? Do try another one that you might be interested in and see if it works! :-)

**Comment by ItzmeJan** [comment]
man i got the same one , i didn't even apply on any

**Reply by tamaraiselvan** [reply]
Leave that particular Octernship, and try to apply for another Octernship. I was able to apply for another Octernship.
